internal/core/core.go

```diff
@@ -99,6 +99,19 @@ type Array struct {
 	Shape []int `json:"shape" swaggertype:"array,integer" example:"10,50"`
 }
 
+// @Description Slice bounds.
+type Bound struct {
+	// Direction of the bound. See SliceRequest.Direction for valid options
+	Direction *string  `json:"direction" binding:"required" example:"inline"`
```

Comment:
Neh, but I was too lazy to add a single test to main_test. Will do. This type of error isn't too bad, because it hits you straight away and will quickly be fixed. Shame on me still tho